jetstream/concepts/consumers.md

```diff
@@ -2,33 +2,115 @@
-| AckWait | How long to allow messages to remain un-acknowledged before attempting redelivery |
-| DeliverPolicy | The initial starting mode of the consumer, `DeliverAll`, `DeliverLast`, `DeliverNew`, `DeliverByStartSequence` or `DeliverByStartTime` |
-| DeliverySubject | The subject to deliver observed messages. Useful to set up an alternate subject for a regular NatsSubcriber can listen on that subject. Not allowed for pull subscriptions. |
-| Durable | The name of the Consumer, which the server will track, allowing resuming consumption where left off. See [naming](../administration/naming.md). |
-| FilterSubject | When consuming from a Stream with many subjects, or wildcards, select only a specific incoming subjects, supports wildcards |
-| MaxDeliver | Maximum amount times a specific message will be delivered.  Use this to avoid poison pills crashing all your services forever |
-| OptStartSeq | When first consuming messages from the Stream start at this particular message in the set |
-| ReplayPolicy | How messages are sent `ReplayInstant` or `ReplayOriginal` |
-| SampleFrequency | What percentage of acknowledgements should be samples for observability, 0-100 |
-| OptStartTime | When first consuming messages from the Stream start with messages on or after this time |
-| RateLimit | The rate of message delivery in bits per second |
-| MaxAckPending | The maximum number of messages without acknowledgement that can be outstanding, once this limit is reached message delivery will be suspended |
+### AckPolicy
+How messages should be acknowledged. The server will consider an ack ony if it comes within the Ack Wait window.
+If the ack is not received in time, the message(s) will be redelivered.
 
+#### AckExplicit
+
+This is the default policy. It means that each individual message must be acknowledged. It is the only allowed option for pull consumers.
+
+#### AckNone
+
+You do not have to ack any messages, the server will assume ack on delivery.
+
+#### AckAll
+
+If you receive a series of messages, you only have to ack the last one you received.
+All the previous messages received are automatically acknowledged.
+
+### AckWait
+
+Ack Wait is the time in nanoseconds that the server will wait for an ack for any individual message.
+If an ack is not received in time, the message will be redelivered.
+
+
+### DeliverPolicy / OptStartSeq / OptStartTime
+
+When a consumer is first created, it can specify where in the stream it wants to start receiving messages.
+This is the `DeliverPolicy` and it's options are as follows:
+
+#### DeliverAll
+
+All is the default policy. The consumer will start receiving from the earliest available message.
+
+#### DeliverLast
+
+The consumer will start receiving messages with the last message added to the stream, so the very last message in the stream when the server realizes the consumer is ready.
+
+#### DeliverNew
+
+The consumer will only start receiving messages that were created after the consumer was created.
```

Comment:
These can be qualified with `When first consuming messages...` also, since if its a durable you will only get from last on the first fetch after that you get what you didnt get before, same for Last and All

Reply:
done